### PR TITLE
chore(refactor): refactor to document title using vue composition api

### PR DESCRIPTION
**Pull request by vimercati-samir**
What this PR does 📖
Reopened AP-1157 to use Vue composition API for document title

Which issue(s) this PR fixes 🔨
AP-1157

Special notes for reviewers 🗒️

Additional comments 🎤

### CODE REVIEW

**Comment by molimauro** [comment]
@vimercati-samir as already mentioned lets leverage the power of the comp api by deleting the mixins and calling the useHead where needed ;) thanks

**Comment by vimercati-samir** [comment]
> @vimercati-samir as already mentioned lets leverage the power of the comp api by deleting the mixins and calling the useHead where needed ;) thanks

Addressed in commit ee9eae15e4cfd0c02228e624c9e2895a28754b53

File: pages/setup/privacy/index.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { validURL } from '~/libraries/ui/Common'
 export default Vue.extend({
   name: 'PrivacyScreen',
   computed: {
-    ...mapState(['settings']),
+    ...mapState(['settings', 'meta']),
```
Comment (molimauro) [inline pages/setup/privacy/index.vue:11]: this is needed?

Comment (vimercati-samir) [reply pages/setup/privacy/index.vue]: Nope, it was leftover code, thanks for reporting it, resolved in commit c0cc54cb3ace892327e0fce21664299274ece3f1

**Comment by molimauro** [comment]
Its not possible to test, just be sure at least that the document.title is setup in all our pages. Thanks!

image

**Comment by stavares843** [comment]
thanks @molimauro 🔨

**Comment by stavares843** [comment]
https://user-images.githubusercontent.com/29093946/163485984-b364f236-b316-4c16-aa54-e5138e2b185d.mov

is this expected when refreshing? showing satellite-im and then satellite-absolute? cc @molimauro 

otherwise, only missing fixing conflict 🔨

**Comment by molimauro** [comment]
> Gravacao.do.ecra.2022-04-14.as.23.15.59.mov 
> is this expected when refreshing? showing satellite-im and then satellite-absolute? cc @molimauro
> 
> otherwise, only missing fixing conflict 🔨

Yes, i have updated "Satellite-Absolute" to "Satellite.im" also in the store, i think the former is deprecated. However when in the future we'll change the title, in the initial app loading we'll still see "Satellite.im" ;)

@vimercati-samir i've updated the `useMeta` function with a computed prop so that when we change the store.title it will be immediately reflected.

**Comment by stavares843** [comment]
agree 🔨 thanks for updating 🎉

**Review by stavares843 — APPROVED** [review]

**Pull request merged**